Khaled JS3 Week3

KhaledAlsaleh opened the pull request.

jasperkennis commented on hackyourrepo-app/script.js:
```diff
@@ -1,5 +1,23 @@
-  Write here your JavaScript for HackYourRepo!
-*/
+import * as dom from './util/domModule.js';
+import {getReposNames,sortReposNames,createOptions,linkRepoNameWithOwnObject,createBodyContributors} from './util/logicAPIModule.js';
```
I think you are importing a bunch of functions here that you are not using in this script. Try to only import what you use.

KhaledAlsaleh replied:
Thanks for your time, I will keep that in mind 👍👍

jasperkennis commented on hackyourrepo-app/util/paginationModule.js:
```diff
@@ -0,0 +1,176 @@
+import {containerPagination,previousButtonPagination,nextButtonPagination} from './domModule.js';
+import {createBodyContributors} from './logicAPIModule.js'
+
+export let itemsTamplet = [];
```
Do you mean template? There is a spelling check available for most editors. For example, I use [code spelling checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) with visual studio code.

KhaledAlsaleh replied:
Yes I made a mistake 😅 I will add it to Visual Studio, thanks a lot.

jasperkennis reviewed (approved):
Hi Khaled,

Great work! I added two small comments, and in general I have these two remarks:

- Your code is a little bit messy in terms of indentation/spacing. Consider using a linter to automatically get your code structure consistent. This is not mandatory but when you start to work in teams you will see that there is usually an agreed standard of codestyling.
- I always like to suggest using only classes for styling, and ids for your logic.

All in all these are minor things and suggestions, no need to make changes per se but I hope you learn something from them😊

Cheers!  Jasper

KhaledAlsaleh commented:
> Hi Khaled,
> 
> Great work! I added two small comments, and in general I have these two remarks:
> 
> - Your code is a little bit messy in terms of indentation/spacing. Consider using a linter to automatically get your code structure consistent. This is not mandatory but when you start to work in teams you will see that there is usually an agreed standard of codestyling.
> - I always like to suggest using only classes for styling, and ids for your logic.
> 
> All in all these are minor things and suggestions, no need to make changes per se but I hope you learn something from them😊
> 
> Cheers!  Jasper

Hello Jasper,

Thanks for your time and tips.

I used linter before but I changed the laptop 😅... I will add it again to VS.

I will use only classes for styling I  future projects.. Thank you very much 🙏

Khaled.
